libimage/load_test.go

```diff
@@ -33,6 +33,8 @@ func TestLoad(t *testing.T) {
 		{"testdata/oci-name-only.tar.gz", false, 1, []string{"localhost/pretty-empty:latest"}},
 		{"testdata/oci-non-docker-name.tar.gz", true, 0, nil},
 		{"testdata/oci-registry-name.tar.gz", false, 1, []string{"example.com/empty:latest"}},
+		// {"testdata/oci-two-names.tar.xz", false, 1, []string{"localhost/pretty-empty:latest", "example.com/empty:latest"}},
```

Comment:
Why is that commented out?

Reply:
Ah that was before I realized we couldn't do two names for oci-archive yet, removed it for now. Will add it back with the implementation to support 2 or more names with oci-archive.